Makefile

```diff
@@ -27,7 +27,7 @@ lint:
 	poetry run mypy .
 	poetry run black . --check
 	poetry run isort . --check
```

Comment:
If you enable `isort`, you can remove this.

Reply:
Then you can change your `poetry run isort .` under `format` to:

```
poetry run ruff --select I --fix .
```

Reply:
nice